xinetd_time.patch

```diff
@@ -0,0 +1,241 @@
+diff --git a/src/access.c b/src/access.c
+index 0a2d271..3517d68 100644
```

Comment:
Was adding this patch (which looks like it has all the changes in this PR) intentional?

Reply:
what do you think I need modify the patch?

Reply:
I think you don't need to add the `xinetd_time.patch` file at all - basically it's a duplicate of the changes you did to the other files.

Reply:
ping @cboltz !